symbi/posts/views.py

```diff
@@ -43,3 +43,10 @@ def createPost(request):
 def deletePost(request, post_id):
     ActivityPost.objects.filter(pk=post_id).delete()
     return HttpResponseRedirect(reverse('posts:home'))
+
+def archivePost(request, post_id):
+    currentPost = ActivityPost.objects.filter(pk=post_id)[0]
+    currentPost.status = 3
+    currentPost.save()
```

Comment:
We might want to check whether the post corresponding to the given `post_id` exists or not here:
```suggestion
    currentPost = ActivityPost.objects.filter(pk=post_id)
    if currentPost.exists():
        currentPost = currentPost.get()
        currentPost.status = 3
        currentPost.save()
```